dashboard/app/helpers/levels_helper.rb

```diff
@@ -182,6 +182,17 @@ def app_options
       # currently, all levels are Blockly or DSLDefined except for Unplugged
       @app_options = view_options.camelize_keys
     end
+
+    @app_options[:dialog] = {
+      skipSound: !!(@level.properties['options'].try(:[], 'skip_sound')),
+      preTitle: @level.properties['pre_title'],
+      fallbackResponse: @fallback_response.to_json,
+      callback: @callback,
+      app: @level.type.underscore,
```

Comment:
Hmm, I'll look a bit closer then. Maybe I'll just keep using the hardcoded values but take it out of appOptions.dialog.